Linux/flash

```diff
@@ -207,11 +212,8 @@ while true; do
     esac
   done
 
-  # paranoia sync before umounting
-  echo "Synchronizing Filesystems"
-  sudo sync; sudo sync; sudo sync
-
-  writable=$(grep "${disk}" /etc/mtab | grep -e "\( rw,\|,rw\)")
+  $(sudo test -w "${disk}")
```

Comment:
Remounting the SD card without the protection is more dangerous. I would prefer to let the user unplug it and move the slider as on Mac.

I found another way which seems promising:

```
vagrant@vagrant:~$ hdparm -r /dev/sdb
/dev/sdb: Permission denied
vagrant@vagrant:~$ sudo hdparm -r /dev/sdb

/dev/sdb:
 readonly      =  1 (on)
vagrant@vagrant:~$ umount /dev/sdb1
vagrant@vagrant:~$ umount /dev/sdb2
vagrant@vagrant:~$ sudo hdparm -r /dev/sdb

/dev/sdb:
 readonly      =  1 (on)
vagrant@vagrant:~$ umount /dev/sdb1
vagrant@vagrant:~$ umount /dev/sdb2
vagrant@vagrant:~$ sudo hdparm -r /dev/sdb

/dev/sdb:
 readonly      =  0 (off)
```